### ISSUE TITLE
Does Linphone ios sdk self send Push notification to APNS?

### ISSUE THREAD

**Issue by ashwani280496** [BelledonneCommunications/linphone-iphone#745]
I am implementing a SIP Calling client .
incoming, outgoing calls are working fine when app is in foreground
now I need to push notification when app is in background. so I want to know does linphone sdk self sends push notifications to APNS? If yes how can i configure this in my code?

Thanks

**Comment by chaoshwu** [comment]
yes, linphone support the PUSH, actually, it's the push notification from APNS to your linphone client as i understand, but you need to prepare your own voip certificate in your apple developer account.

**Comment by Raviojha62** [comment]
@ashwani280496  you need to setup voip notification on your server side for background or terminated state , Linphone support for foreground only
